app/assets/javascripts/discourse/app/controllers/preferences/profile.js

```diff
@@ -57,10 +57,6 @@ export default Controller.extend({
     "model.can_upload_user_card_background"
   ),
 
-  experimentalUserCardImageUpload: readOnly(
-    "siteSettings.enable_experimental_image_uploader"
```

Comment:
Do we still need the site setting on the server side? If not, we'll need to drop it from the database.